### DISCUSSION TITLE
Kairos is very useful. Sometimes, I want to disable the read-only system so that I can install tools such as packet capture and Git for testing. Is there any way to disable the read-only system

### DISCUSSION THREAD

**Discussion by cheng990629** [kairos-io/kairos#2323] [General]
Kairos is very useful. Sometimes, I want to disable the read-only system so that I can install tools such as packet capture and Git for testing. Is there any way to disable the read-only system

**Comment by BlackGrassland** [comment]
rd.cos.disable/rd.immucore.disable on the cmdline
